libsweep/src/protocol.cc

```diff
@@ -2,10 +2,15 @@
 #include <stdlib.h>
 
 #include "protocol.h"
+#include "../include/Error.hpp"
```

Comment:
Please no relative include paths, the include dir should be on the search path already (check CMakeLists.txt)